### PR TITLE
Update some of the code behind with what's on the VM

### PR DESCRIPTION
**Pull request by twsouthwick**
These are some changes from the VM that will fix some of the issues we've seen with some of the data (ie DownloadBitmap in MarsHirise).

### CODE REVIEW

File: WWTMVC5/WWTWeb/MarsMoc.aspx [hunk 1]
```diff
@@ -22,7 +23,7 @@
     Bitmap bmp1 = null;
     Bitmap bmp2 = null;
 
-    if (level > 15)
+    if (level > 14)
```
Comment (pkgw) [inline WWTMVC5/WWTWeb/MarsMoc.aspx:26]: We should make sure to confirm that the maximum tile level in this plate file is in fact 14.

File: WWTMVC5/WWTWeb/MartianTile.aspx [hunk 2]
```diff
@@ -9,76 +9,64 @@
...
-			break;
-		case "mars_historic_lowell":
-			id = "675790761";
-			break;
-		case "mars_historic_antoniadi":
-			id = "1648157275";
-			break;
-		case "mars_historic_mec1":
-			id = "2141096698";
-			break;
```
Comment (pkgw) [inline WWTMVC5/WWTWeb/MartianTile.aspx:51]: If I'm reading this correctly, we're losing support for most of the other Mars datasets here. They're not heavily used so this is OK in the interim but this the kind of thing we should write down and make sure to patch up later.

File: WWTMVC5/WWTWeb/MartianTile.aspx [hunk 3]
```diff
@@ -9,76 +9,64 @@
...
-        //catch
-       // {
-            Response.StatusCode = 404;
-            return;
-       // }
-    }
+
+    WebClient client = new WebClient();
+
+    byte[] data2 = client.DownloadData("http://141.142.209.200/wwtweb/martiantile.aspx?q="+query);
```
Comment (pkgw) [inline WWTMVC5/WWTWeb/MartianTile.aspx:66]: Ah OK, looks like we're hardcoding a fallback to the main server? OK as a stopgap but there should at least be a big "XXX FIXME" comment as a reminder ...

File: WWTMVC5/WWTWeb/demmars.aspx [hunk 4]
```diff
@@ -1,47 +1,46 @@
...
-       //     webclient.DownloadFile(url, filename);
-      //  }
-      //  catch
-       // {
-            Response.StatusCode = 404;
+    //    Response.ContentType = "image/png";
+
+        UInt32 index = ComputeHash(level, tileX, tileY) % 400;
+
+        Stream s = PlateFile2.GetFileStream(String.Format(@"\\wwtfiles.file.core.windows.net\wwtmars\MarsDem\marsToastDem.plate", index), -1, level, tileX, tileY);
```
Comment (pkgw) [inline WWTMVC5/WWTWeb/demmars.aspx:26]: `index` isn't used in the `Format` here?

File: WWTMVC5/WWTWeb/GetTours.aspx.cs [hunk 5]
```diff
@@ -1,191 +1,23 @@
...
+using System.Web.UI;
+using System.Web.UI.WebControls;
+using System.Web.UI.WebControls.WebParts;
+using System.Web.UI.HtmlControls;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
 using System.Xml;
 using WWTWebservices;
+using Microsoft.Research.WWT;
```
Comment (pkgw) [inline WWTMVC5/WWTWeb/GetTours.aspx.cs:17]: Compile error triggered here.

File: WWTMVC5/WWTWeb/MarsMoc.aspx (hunk 1 above)
Comment (twsouthwick) [reply WWTMVC5/WWTWeb/MarsMoc.aspx:26]: How do we check that? This is the code running on the VM right now, so I figured it had the appropriate levels.

File: WWTMVC5/WWTWeb/MartianTile.aspx (hunk 3 above)
Comment (twsouthwick) [reply WWTMVC5/WWTWeb/MartianTile.aspx:66]: That's weird - I thought this was the "main" server? Is that not pointing to the VM? Which also would be weird, but I must be missing something here.

File: WWTMVC5/WWTWeb/demmars.aspx (hunk 4 above)
Comment (twsouthwick) [reply WWTMVC5/WWTWeb/demmars.aspx:26]: Good catch. Appears to be so. Once we get the code moved out of the aspx files, then things like that can be caught very easily by some new tooling built into .NET.

File: WWTMVC5/WWTWeb/MarsMoc.aspx (hunk 1 above)
Comment (pkgw) [reply WWTMVC5/WWTWeb/MarsMoc.aspx:26]: The most convenient thing that I can think if is to run the code with the level 15 check, then try to request a level 15 tile and see if it works.

Really these kinds of limits shouldn't be hardcoded into the ASPX, but should instead come from the actual data file ... probably the existing code crashes if you go too deep, or something.

File: WWTMVC5/WWTWeb/MartianTile.aspx (hunk 3 above)
Comment (pkgw) [reply WWTMVC5/WWTWeb/MartianTile.aspx]: Mystery solved, sort of — I didn't realize this was more code from the VMs directories. No idea what this magic IP address is/was, but we're not gonna want it anymore ...

File: WWTMVC5/WWTWeb/MarsMoc.aspx (hunk 1 above)
Comment (twsouthwick) [reply WWTMVC5/WWTWeb/MarsMoc.aspx:27]: Added a comment to that effect to look into that

**Comment by twsouthwick** [comment]
@pkgw Updated per our conversation

**Pull request merged**